src/lib/bootloader/grub2_widgets.rb

```diff
@@ -1115,5 +1116,15 @@ def contents
         VStretch()
       )
     end
+
+  private
+
+    def os_prober_widget
+      if OsProber.available? # Checks !Arch.s390 and if package is available
+        Left(OSProberWidget.new)
+      else
+        CWM::Empty.new("os_prober")
```

Comment:
Originally, there was a Left() around it; does it make a difference?

Reply:
No, empty is empty, no matter if it's left-aligned or not. ;-)